.gitignore

```diff
@@ -1,3 +1,6 @@
+.vscode
+*.xml
```

Comment:
Are there more appropriate places to put these lines, than right at the top of this file?

Reply:
Possibly but not sure where you would like.

Reply:
Done.